source/client/process_impl.cc

```diff
@@ -179,7 +181,10 @@ void ProcessImpl::createWorkers(const uint32_t concurrency) {
   // TODO(oschaaf): Arguably, this ought to be the job of a rate limiter with awareness of the
   // global status quo, which we do not have right now. This has been noted in the
   // track-for-future issue.
-  const auto first_worker_start = time_system_.systemTime() + kMinimalWorkerDelay;
+  const Envoy::MonotonicTime monotonic_now = time_system_.monotonicTime();
+  const std::chrono::nanoseconds offset =
+      schedule.has_value() ? schedule.value() - time_system_.systemTime() : kMinimalWorkerDelay;
```

Comment:
3. We are heavily explaining the workerdelay behavior here in the comment above.  Can we add a one-line comment for how the schedule behavior fits in?  For instance, if I am wrong about point 2 above, we should say why the scheduled workers don't need an offset delay.